whitenoise/base.py

```diff
@@ -1,10 +1,7 @@
-from __future__ import absolute_import
```

Comment:
@timgraham one stupid question that is unrelated to this: some technical inconvenient who takes the need for removal of `from __future__ import absolute_import`